lib/rules/selector-pseudo-class-no-unknown/index.js

```diff
@@ -98,6 +99,10 @@ const rule = function (actual, options) {
       let selector = null
 
       if (node.type === "rule") {
+        if (!isStandardSyntaxRule(node)) {
+          return
+        }
+
         selector = node.selector
       } else if (node.type === "atrule" && node.name === "page" && node.params) {
```

Comment:
Use `isStandardSyntaxAtRule` here for completeness - https://github.com/stylelint/stylelint/blob/a756b20559d80a1bbef2f555215370199757ee34/lib/rules/selector-pseudo-class-no-unknown/index.js#L108-L110